### ISSUE TITLE
Getting error 'Could not find generator 'spree:frontend:copy_storefront'

### ISSUE THREAD

**Issue by developeritsacheckmate** [spree/spree#9927]
Getting error 'Could not find generator 'spree:frontend:copy_storefront'

## Context
Use this
```
gem 'spree', '~> 4.0'
gem 'spree_auth_devise', '~> 4.0'
gem 'spree_gateway', '~> 3.6'
```
And execute the command mentioned at https://guides.spreecommerce.org/developer/customization/view.html

rails g spree:frontend:copy_storefront

## Expected Behavior
no error

## Actual Behavior
Could not find generator 'spree:frontend:copy_storefront'. Maybe you meant "spree:frontend:copy_views"?

## Possible Fix
Update readme?

**Comment by pkurek** [comment]
I'm having the same issue. It seems like the whole readme is outdated and not well tested. I just set up a fresh rails 6 project copy/paste their gems (from readme) and I'm getting gem version errors when trying to install through bundler :)

**Comment by damianlegawiec** [comment]
@developeritsacheckmate that command is only available for Spree 4.1, please update your version

**Comment by damianlegawiec** [comment]
> I'm having the same issue. It seems like the whole readme is outdated and not well tested. I just set up a fresh rails 6 project copy/paste their gems (from readme) and I'm getting gem version errors when trying to install through bundler :)

https://github.com/spree/spree#install-gems

<img width="802" alt="Screenshot 2020-09-14 at 10 55 41" src="https://user-images.githubusercontent.com/55154/93065313-d64b7800-f678-11ea-8870-ebf56e06cd5d.png">
